### ISSUE TITLE
Error with Builcraft (compat) 1.7.10

### ISSUE THREAD

**Issue by nicogranelli** [rubensworks/EvilCraft#374]
Relevant part:

```
java.lang.ClassCastException: evilcraft.fluid.Blood cannot be cast to net.minecraftforge.fluids.FluidStack
    at buildcraft.compat.nei.RecipeHandlerBase.loadCraftingRecipes(RecipeHandlerBase.java:76)
```

Not sure if is in your side.

Version: EvilCraft-1.7.10-0.9.7

Complete log: http://pastebin.com/MeNf0Pa3

**Comment by rubensworks** [comment]
This may be caused by EC, I'll have to look into it.

**Comment by nicogranelli** [comment]
I forgot to say: this happened when I clicked in the NEI handler for a EC machine. I clicked in the blood part.

**Comment by rubensworks** [comment]
Yeah, I expected something like that. I'm probably passing the wrong parameter somewhere. I hope to look into it somewhere this weekend.

**Comment by immortaleeb** [comment]
The issue should be fixed in the next 1.7.10 release, which you can probably expect within a week or so
